build_latest.sh

```diff
@@ -107,7 +107,6 @@ for vm in ${all_jvms}
 do
 	for package in ${all_packages}
 	do
-		oses=$(parse_os_entry ${vm})
```

Comment:
Was that deliberate?

Reply:
Yes, Both changes are needed. This was added by mistake in the JRE patch. Need to remove this.